notebooks/guaranteeing_valid_syntax.ipynb

```diff
@@ -191,7 +220,9 @@
     "        \"mantra\" : { \"type\" : \"string\" },\n",
     "        \"strength\" : { \"type\" : \"integer\" },\n",
     "        \"quest_items\" : { \"type\" : \"array\", \"items\" : { \"type\" : \"string\" } }\n",
-    "    }\n",
+    "    },\n",
+    "    \"required\": [ \"description\", \"name\", \"age\", \"armour\", \"weapon\", \"class\", \"mantra\", \"strength\", \"quest_items\" ],\n",
```

Comment:
This seriously comes under the heading of "This does not work how you think it works" :-/